### DISCUSSION TITLE
Is there a user interface to list all services with ports in Ocelot?

### DISCUSSION THREAD

**Discussion by rifaterdemsahin** [ThreeMammals/Ocelot#1982] [Q&A]
I am coming from Consul and I expect a UI to be able to see the orchestration.

Does Ocelot have UI similar to Consul ?

**Comment by raman-m** [comment]
No! Ocelot has no any UI, no monitoring feature!
Ocelot has no orchestration feature too.
Ocelot behaves as backend service.
To understand you better we need to agree on the terms: UI, orchestration.
